### PR TITLE
Update dm upgrade documentation

### PR DESCRIPTION
**Pull request by liubo0127**
### What problem does this PR solve?
Resolve version incompatibility issues after upgrade.
@WangXiangUSTC @GregoryIan PTAL

### CODE REVIEW

File: docs/dm/zh_CN/maintenance/dm-ansible.md [hunk 1]
```diff
@@ -340,21 +340,25 @@ $ ansible-playbook stop.yml
...
+    $ mv dm-ansible-latest dm-ansible
+    ```
 
-2. 使用 playbook 下载最新的（latest） DM binary，自动替换 `/home/tidb/dm-ansible/resource/bin/` 目录下 binary 文件。
-```
-$ ansible-playbook local_prepare.yml
-```
+3. 使用 playbook 下载最新的（latest） DM binary。
 
-> 如果需要根据 Git Commit Hash 下载指定版本的 DM binary，可以直接从 http://pingcap-dev.hk.ufileos.com/builds/pingcap/tidb-enterprise-tools/{git-commit-hash}/centos7/dm-linux-amd64.tar.gz 进行下载。
```
Comment (csuzhangxc) [inline docs/dm/zh_CN/maintenance/dm-ansible.md:356]: should we need to support download specified version of DM through DM-Ansible?

**Review by csuzhangxc — COMMENTED** [review]
rest LGTM

File: docs/dm/zh_CN/maintenance/dm-ansible.md (hunk 1 above)
Comment (IANTHEREAL) [reply docs/dm/zh_CN/maintenance/dm-ansible.md:356]: ping @liubo0127 we have `v1.0 alpha` now

**Comment by IANTHEREAL** [comment]
LGTM

**Comment by IANTHEREAL** [comment]
LGTM

File: docs/dm/zh_CN/maintenance/dm-ansible.md [hunk 2]
```diff
@@ -105,17 +105,27 @@ The key's randomart image is:
 
 以 `tidb` 用户登录中控机并进入 `/home/tidb` 目录。
 
-```
-$ wget http://download.pingcap.org/dm-ansible-latest.tar.gz
-```
+- 下载 1.0.0-alpha 版本
+
+    ```
+    $ wget https://download.pingcap.org/dm-ansible-v1.0.0-alpha.tar.gz
```
Comment (csuzhangxc) [inline docs/dm/zh_CN/maintenance/dm-ansible.md:111]: in *dm-ansible-v1.0.0-alpha.tar.gz*, we still have `dm_version = latest`.
so, should we need to update this `dm_version` in *dm-ansible-v1.0.0-alpha.tar.gz* and re-release it?

Comment (csuzhangxc) [reply docs/dm/zh_CN/maintenance/dm-ansible.md:111]: ping @liubo0127

Comment (IANTHEREAL) [reply docs/dm/zh_CN/maintenance/dm-ansible.md:111]: we should do it ASAP

**Comment by IANTHEREAL** [comment]
I would close this pr, because we don't need this document any more

**Pull request closed without merging**